### PR TITLE
RC 1.6.0

### PR DESCRIPTION
**Pull request by ramirotw**
# Summary

This minor release added small improvements to existing features as well as some refactors.

# Changelog

320cd47ed Enable 1inch (#1942)
2fc6c8187 1334/activity orders from api rebased (#1639)
86635055b Turning on thousand separator for FIAT values (#1934)
e97277fc0 Change typo (#1926)
c01516860 Version images to prevent hard caches (#1925)
37518ffaa Add class and id for easy e2e (#1922)
de6e1fa66 Increase update time for orders (#1893)
443a98f71 Filter emtpy queryParams (#1881)
443ecf0a9 [Affiliate] Update banner closed state across tabs (#1875)
02fd6cb53 Handle cancel state for ethereum txs (#1532)
857316230 (bugfix) Terms and conditions styles (#1894)
28f8173bc bold numbers in T&C page (#1860)
738006cc0 [Profile] Add name to IPFS files + Upload JSON using pinata (#1865)
3a6753463 Fix image link in README (#1838)

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @ramirotw , @alfetopito , should we include into the RC #1639 pr?
As far as I understand, the feature is still under implementation (see #1951)

Thanks

**Comment by elena-zh** [comment]
Hey @ramirotw , I can't get 'Cancelling' state for transactions in this PR (#1532 ) for the Mainnet transactions. See the video: https://watch.screencastify.com/v/IefumKIH4F3heceqAEw1

The same is for the Wrap and Unwrap transactions.

**Comment by ramirotw** [comment]
> Hey @ramirotw , @alfetopito , should we include into the RC #1639 pr? As far as I understand, the feature is still under implementation (see #1951)
> 
> Thanks

Good call, not sure, @alfetopito created the `release/1.6.0` branch which already included that commit, I didn't change the history

**Comment by elena-zh** [comment]
> > Hey @ramirotw , @alfetopito , should we include into the RC #1639 pr? As far as I understand, the feature is still under implementation (see #1951)
> > Thanks
> 
> Good call, not sure, @alfetopito created the `release/1.6.0` branch which already included that commit, I didn't change the history

Anyways, it works as expected: orders are synced and loaded from API.

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "description": "CowSwap - Gnosis Protocol",
   "homepage": ".",
   "private": true,
-  "version": "1.5.1",
+  "version": "1.6.0-rc.0",
```
Comment (alfetopito) [inline package.json:6]:
```suggestion
  "version": "1.6.0",
```

**Review by alfetopito — APPROVED** [review]

**Comment by ramirotw** [comment]
> Hey @ramirotw , I can't get 'Cancelling' state for transactions in this PR (#1532 ) for the Mainnet and xDAI transactions. See the video: https://watch.screencastify.com/v/IefumKIH4F3heceqAEw1

@elena-zh is the Speed up (and transaction replacement) working? just to be sure the blocknative integration works for mainnet

**Comment by elena-zh** [comment]
@ramirotw , seems that no, but I'm not sure. See the video: 
https://watch.screencastify.com/v/INV1huBlsBwzmTuVUNek

**Comment by ramirotw** [comment]
> @ramirotw , seems that no, but I'm not sure. See the video:
> https://watch.screencastify.com/v/INV1huBlsBwzmTuVUNek

did the Speed up transaction get mined?

**Comment by alfetopito** [comment]
Elena, without any deep investigation, I suspect this could be due to the race condition in case the original tx gets mined before the speed-up tx. If so, it'll be addressed on that task.

**Comment by elena-zh** [comment]
> > @ramirotw , seems that no, but I'm not sure. See the video:
> > https://watch.screencastify.com/v/INV1huBlsBwzmTuVUNek
> 
> did the Speed up transaction get mined?

Finally, yes. https://etherscan.io/tx/0x6a7028a6a2447483b5d8700024beb42c3b77c6f2c26ded0eb54e8fc644fe135e
TxHash matches in the the wallet and in the app. 

Btw, approve transaction from the 1st video was also mined in an hour or two. It was not cancelled, however, I tried it several times in my MM wallet. https://etherscan.io/tx/0xd96f9e333278e8d64e562259c55e27e7593a3253c1ef2f7423a3f117a07daa75

**Pull request merged**